### ISSUE TITLE
55% of integration tests don't pass

### ISSUE THREAD

**Issue by ProducerMatt** [StampyAI/stampy#284]
In test 5, the ChatGPT module actually intercepts the factoid module, which I found surprising

![image](https://github.com/StampyAI/stampy/assets/58014742/924b6741-34f3-4581-ac57-b2d090603958)![image](https://github.com/StampyAI/stampy/assets/58014742/f6c32471-363b-4cf4-ad00-535bde72ac98)

**Comment by ProducerMatt** [comment]
What is this? Is it supposed to be getting a DDG definition?

```
`TEST #6: FAILED`
The sent message was: "TEST_MESSAGE 6: what is deep learning?"
The expected message was "RegEx: Deep learning is part of a broader family of machine learning method"
The received message was "Perhaps this can answer your question? https://aisafety.info?state=8XBK_"
```

**Comment by Aprillion** [comment]
yup, a test from duckduckgo, you can search for "what is deep learning?" in the code:

https://github.com/StampyAI/stampy/blob/1129a10a2117d2bd37b0b5d41210afa41dd9a8c6/modules/duckduckgo.py#L123-L124

not sure what would be a reliable test that wouldn't be overruled by other modules in the future - feel free to delete fragile tests for now if you won't figure out how to replace them with something more stable 🤔
